### ISSUE TITLE
Show which Task is associated to a given WorkflowTask

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#239]
Reason for this: we currently only show the Task names, in the list of WorkflowTasks, but this could be too generic (e.g. if there are two versions).

This would probably go into WorkflowTask "info" tab.

**Comment by jluethi** [comment]
Agreed, would be great to have some task information in the info tab. The version of the task for sure. Also on whether it's a common or user specific task (of which user?), because different users could have the same name & version.

It may even make sense to expose input & output types of the tasks in the info tab.

And of course the actual task description, once we get it from the task package into Fractal server
